signing_service/signing_service/signing_service.py

```diff
@@ -491,6 +496,20 @@ def _parse_arguments() -> argparse.Namespace:
         help=f'Environment which will be set in Raven client config `environment` parameter.',
     )
 
+    subparsers = parser.add_subparsers(help='sub-command help')
+
+    # create the parser for the console notifier
+    parser_for_local_notifier = subparsers.add_parser('console-notifier', help='default console notifier')
+    parser_for_local_notifier.add_argument('--from-email-address', default=None)
+    parser_for_local_notifier.add_argument('--from-email-password', default=None)
+    parser_for_local_notifier.add_argument('--to-email-addresses', default=[''])
+
+    # create the parser for the email notifier
+    parser_for_cluster_notifier = subparsers.add_parser('smtp-notifier', help='smtp notifier with no harcoded values')
+    parser_for_cluster_notifier.add_argument('from-email-address', type=str, help='gmail login')
+    parser_for_cluster_notifier.add_argument('from-email-password', type=str, help='gmail password')
+    parser_for_cluster_notifier.add_argument('to-email-addresses', type=List[str], help='list of addresses to send norifications to.')
```

Comment:
"norifications" -> "notifications"

Reply:
Done

Reply:
BTW: Make all those arguments required, with `required=True`

Reply:
I just did it actually for `from-email-address` and `from-email-password`, but I'm not sure if `to-email-addresses` should be required too, because leaving it empty don't cause any errors.